include/mbedtls/base64.h

```diff
@@ -26,8 +26,10 @@
 
 #include <stddef.h>
 
-#define MBEDTLS_ERR_BASE64_BUFFER_TOO_SMALL               -0x002A  /**< Output buffer too small. */
-#define MBEDTLS_ERR_BASE64_INVALID_CHARACTER              -0x002C  /**< Invalid character in input. */
+#define MBEDTLS_ERR_BASE64_BUFFER_TOO_SMALL \
+    -0x002A /**< Output buffer too small. */
```

Comment:
This is looks horrible. Is there any way in the tool options to define that the line break will happen in the comment rather than in the code? Or to move the comment above the code line?

Reply:
see my comment above

Reply:
Ah, OK, didn't see.

Reply:
Basically the clang-format is quite stubborn in this matter. I was not able to persuade to leave the assigned value as it was and adjust only the comment.

Reply:
As suggested for ssl.h, we could change them manually before to reformat automatically.